### DISCUSSION TITLE
[Help]: How to use image captions

### DISCUSSION THREAD

**Discussion by albinlang** [davidjerleke/embla-carousel#913] [Help] [answered]
### Summary

Hi,

Is it possible to include image captions in the carousel?
In my case I'd like to add a caption under each image to improve accessibility.

I read through the documents but I might have missed something 😅 Best case there's an example.

Thank you for a nice project!

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by meirroth** [comment, marked as answer]
Hey @albinlang, you can pretty much put anything in the carousel slide, divs images videos captions you name it. Did you run into an issue when trying?
